src/components/App/styledComponents.styles.ts

```diff
@@ -196,6 +200,8 @@ export const StyledCard = styled.section<StyledCardProps>`
     height: ${props => {
       if(props.contents === 'list') {
         return '75%;'
+      } else if(props.contents === 'fullPage') {
+        return '25%;'
       } else {
         return '80%;'
```

Comment:
This is such a cool way to make the design responsive!

Reply:
Styled components are awesome!